projects/systelab-translate/src/lib/systelab-translate.module.ts

```diff
@@ -7,34 +7,51 @@ import { NumberFormatPipe } from './number-format.pipe';
 import { DecimalPipe, Location, LocationStrategy, PathLocationStrategy } from '@angular/common';
 
 // AoT requires an exported function for factories
-export function httpLoaderFactory(http: HttpClient, location: Location) {
-	return new LocalizableTranslateStaticLoader(http, location);
+export function httpLoaderFactory(http: HttpClient, location: Location, url: string) {
+  return new LocalizableTranslateStaticLoader(http, location, url);
 }
 
+export interface ISystelabTranslateModule {
```

Comment:
Do not use I prefix. We do not have that code convention.

Rename the interface to something like TranslationConfiguration

Make the customUrl optional.